src/detectors/BHCAL/BHCAL.cc

```diff
@@ -23,6 +23,7 @@ extern "C" {
           {
             .eRes = {},
             .tRes = 0.0 * dd4hep::ns,
+            .threshold = 5.0 * dd4hep::MeV,
```

Comment:
There is a prior work https://github.com/eic/EICrecon/pull/674 which took another approach of setting threshold on ADC quantity at the hit reconstruction instead of digitization.

I don't quite understand why they went with `thresholdFactor` instead of `thresholdValue`. Those are interchangeable, but latter doesn't involve the expected pedestal sigma:
https://github.com/eic/EICrecon/blob/0218bf17021eea7586254fb1be571ad2bedbb699/src/algorithms/calorimetry/CalorimeterHitReco.cc#L25 I do agree with their approach more, for the reasons mentioned in https://chat.epic-eic.org/main/pl/szkezeezwjnb3fiaiihg8ze87o

This one should be close to being equivalent. I do have some questions to the implementation of this parameter. Look at it:
https://github.com/eic/EICrecon/blob/2c7550a21588f59bf889253f32a0dfeb43ccdeaa/src/algorithms/calorimetry/CalorimeterHitDigi.cc#L144-L152
It does not physically removes the hit, just zeroes the physical energy contribution and keeps the contribution of noise in place. I don't understand what that is supposed to model.

Reply:
> just zeroes the physical energy contribution and keeps the contribution

Just guessing, but zero acts like an overflow bin then?

Reply:
> > just zeroes the physical energy contribution and keeps the contribution
> 
> Just guessing, but zero acts like an ~overflow~underflow bin then?

Formally, the normal distribution has infinite tails, so a `ped` can be arbitrarily large. I guess, it can't surpass a 36-sigma cut in Sc-Fi PR :D